[New Tracker] Fix Partis indexer to work with new website

mjazbc opened the pull request:
Partis.si indexer was removed in https://github.com/Jackett/Jackett/commit/e86eda9ea3dfee9e078296ca08f9a7ef818566fa because tracker was unavailable since March 2021. Website has now been rebuilt and I have managed to reuse the majority of previous indexer implementation. 

Changes:

- Urls for login, logout, search and torrent download
- Username and password fields in login request
- Json is now returned and HTML parsing is not needed anymore

There are also some issues with the new indexer:

- No way to identify freeleech torrents from a single request. Indexer would have to make another request for each hit in search results
- Number of grabs is currently displayed on a 0-100 scale. I could not find a way to get the total number

This PR fixes https://github.com/Jackett/Jackett/issues/1596

garfield69 reviewed (commented):
>Number of grabs is currently displayed on a 0-100 scale

I don't have access to the site so I cannot review this. 
But from experience on other sites, that is probably not a grabs/downloaded value, but a health check presented in a progress bar.
I suggest you remove the release.Grabs altogether.

So have you checked the categories are all still the same numbers and descriptions?
have you tested that you can download (just as far as seeing that the download prompt comes up, no need to actually perform a save of the torrent) ?

you could add torrent[6] to the release.Description, its a string value that gets presented as a hover popup on the Jackett search results page, 
and if you assign torrent[11] to release.Poster which is a uri, then you can display the poster and get the description auto appended to the bottom of the poster ;-)

mjazbc commented:
>  that is probably not a grabs/downloaded value, but a health check presented in a progress bar

Makes sense, it is indeed a progess bar on the website. It's just that the header above it literally says "downloads", so it didn't much sense. Anyway, i have now removed this value from release.Grabs.

I reviewed categories one more time and found a couple of mistakes (Sports and Anime). I also added some new categories. 
Download was tested from Manual search window - click on download icon downloads the correct .torrent file. 

Description and poster fields are now populated aswell.

garfield69 reviewed (approved):
LGTM, albeit untested.
@ilike2burnthing  were you able to get your a/c going?
Don't forget the old site accepted Slovenian IP only, this may continue to be the case.

ilike2burnthing commented:
No such luck. Site is accessible without a Slovenian IP, but I double-checked with one just in case there was some very odd login/recovery block without it - nope.

garfield69 commented:
Same here. I'll merge it anyway.
